LOAN/LoanTypes/LoanProducts.rdf

```diff
@@ -103,30 +123,163 @@
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;LoanMarketCategory">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-cls;IndustrySectorClassifier"/>
+		<rdfs:label>loan market category</rdfs:label>
+		<skos:definition>category indicating the market domain in which the loan product is offered</skos:definition>
+	</owl:Class>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-Automobile">
+		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+		<rdfs:label>auto loan</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-Commercial">
+		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+		<rdfs:label>commercial loan</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-Construction">
+		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+		<rdfs:label>construction loan</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-Education">
+		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+		<rdfs:label>education loan</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-Mortgage">
+		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+		<rdfs:label>mortgage loan</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-Personal">
+		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+		<rdfs:label>personal loan</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-SmallBusiness">
+		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+		<rdfs:label>small business loan</rdfs:label>
+		<fibo-fnd-utl-av:explanatoryNote>This is related to the mortgage loan purpose: &apos;business or commercial&apos;.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:NamedIndividual>
+	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;LoanProduct">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-pas-fpas;FinancialProduct"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-pas-fpas;isEmbodiedIn"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-ln;Loan"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;refersTo"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-dae-dbt;Collateral"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;hasPreconditions"/>
+				<owl:someValuesFrom rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/DebtAndEquities/CreditFacilities/LoanPrecedentConditions"/>
```

Comment:
why not use the XML entity?

Reply:
Again, I just moved it, and fixed those at the higher level, didn't review any of the content in this ontology, just in LoanCore and a little in applications and a little in mortgages.